src/core/qgspluginlayerregistry.h

```diff
@@ -82,7 +82,7 @@ class CORE_EXPORT QgsPluginLayerRegistry
 
     /**
      * List all known layer types
-     *  \since QGIS */
+     */
```

Comment:
@nyalldawson just me curious: looking to the changes, should this be aligned with line 83?

Reply:
There's no consistency there yet -- maybe we need a new script to fix that :)

Reply:
I was wondering if we should not drop astyle in favor of clang-format. Anyone has an opinion here?